core/quorum/api.go

```diff
@@ -0,0 +1,804 @@
+	ErrNotOrgAdmin        = ExecStatus{false, "Operation can be performed by org admin only. Account not a org admin."}
+	ErrNodePresent        = ExecStatus{false, "EnodeId already part of network."}
+	ErrInvalidNode        = ExecStatus{false, "Invalid enode id"}
+	ErrInvalidAccount     = ExecStatus{false, "Invalid account id"}
+	ErrPermissionDisabled = ExecStatus{false, "Permissions control not enabled"}
+	ErrOrgExists          = ExecStatus{false, "Org already exists"}
+	ErrPendingApprovals   = ExecStatus{false, "Pending approvals for the organization. Approve first"}
+	ErrNothingToApprove   = ExecStatus{false, "Nothing to approve"}
+	ErrOpNotAllowed       = ExecStatus{false, "Operation not allowed"}
+	ErrNodeOrgMismatch    = ExecStatus{false, "Enode id passed does not belong to the organization."}
+	ErrBlacklistedNode    = ExecStatus{false, "Blacklisted node. Operation not allowed"}
+	ErrBlacklistedAccount = ExecStatus{false, "Blacklisted account. Operation not allowed"}
+	ErrAccountOrgAdmin    = ExecStatus{false, "Account already org admin for the org"}
+	ErrOrgAdminExists     = ExecStatus{false, "Org admin exists for the org"}
+	ErrAccountInUse       = ExecStatus{false, "Account already in use in another organization"}
+	ErrRoleExists         = ExecStatus{false, "Role exists for the org"}
+	ErrRoleActive         = ExecStatus{false, "Accounts linked to the role. Cannot be removed"}
+	ErrAdminRoles         = ExecStatus{false, "Admin role cannot be removed"}
+	ErrInvalidOrgName     = ExecStatus{false, "Org id cannot contain special characters"}
+	ErrInvalidParentOrg   = ExecStatus{false, "Invalid parent org id"}
+	ErrAccountNotThere    = ExecStatus{false, "Account does not exists"}
+	ErrOrgNotOwner        = ExecStatus{false, "Account does not belong to this org"}
+	ErrMaxDepth           = ExecStatus{false, "Max depth for sub orgs reached"}
+	ErrMaxBreadth         = ExecStatus{false, "Max breadth for sub orgs reached"}
+	ErrNodeDoesNotExists  = ExecStatus{false, "Node does not exists"}
+	ErrOrgDoesNotExists   = ExecStatus{false, "Org does not exists"}
+	ErrInactiveRole       = ExecStatus{false, "Role is already inactive"}
+	ErrInvalidRole        = ExecStatus{false, "Invalid role"}
+	ErrInvalidInput       = ExecStatus{false, "Invalid input"}
+	ErrNotMasterOrg       = ExecStatus{false, "Org is not a master org"}
+
+	ExecSuccess = ExecStatus{true, "Action completed successfully"}
+)
+
+// NewQuorumControlsAPI creates a new QuorumControlsAPI to access quorum services
+func NewQuorumControlsAPI(tp *core.TxPool, am *accounts.Manager) *QuorumControlsAPI {
+	return &QuorumControlsAPI{tp, nil, am, nil, nil, false, nil, nil}
+}
+
+//Init initializes QuorumControlsAPI with eth client, permission contract and org key management control
+func (p *QuorumControlsAPI) Init(ethClnt *ethclient.Client, key *ecdsa.PrivateKey, apiName string, pconfig *types.PermissionConfig, pc *pbind.PermInterface) error {
```

Comment:
please use a consistent and meaningful receiver.
`func (p *QuorumControlsAPI)` vs `func (s *QuorumControlsAPI)`

Suggestion: `function (q *QuorumControlsAPI)`

Reply:
Changed to function (q *QuorumControlsAPI)